tekton/ci/jobs/tekton-python-test.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: tekton.dev/v1beta1
```

Comment:
This file needs to be added to the `kustomize.yaml` in this folder so that it gets deployed